### PR TITLE
Update the disk volume sync usage command

### PR DESCRIPTION
**Pull request by ebelter**

### CODE REVIEW

File: lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm [hunk 1]
```diff
@@ -39,20 +44,24 @@ sub help_detail {
...
-        if ($self->total_kb)      {
-            $self->debug('sync_total_kb');
-            $volume->sync_total_kb(%args);
+    my %args = (verbose => 1);
+    for my $volume ( $self->volumes ) {
+        my $transaction = UR::Context::Transaction->begin;
+        try {
+            $self->info('Syncing volume: '.$volume->mount_path);
+            if ($self->total_kb)      {
+                $self->debug('Sync total kb...');
```
Comment (nnutter) [inline lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm]: s/kb/kB/

File: lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm [hunk 2]
```diff
@@ -39,20 +44,24 @@ sub help_detail {
...
+            }
+            $transaction->commit or die 'Failed to commit volume! '.$volume->mount_path;
         }
-        if ($self->unallocated_kb) {
-            $self->debug('sync_unallocated_kb');
-            $volume->sync_unallocated_kb(%args);
+        catch {
+            $self->error($_);
+            $transaction->rollback;
         }
```
Comment (nnutter) [inline lib/perl/Genome/Disk/Command/Volume/SyncUsage.pm]: Might be good to terminate with a semicolon.

**Comment by nnutter** [comment]
:+1:

**Comment by acoffman** [comment]
+1

**Pull request merged**